### PR TITLE
Clouddst-14552 Set up Github actions checks for CGW

### PR DESCRIPTION
**Pull request by jalam453**
- Setting up GitHub actions for CGW
    - Updating formatting for black and flake8
    - Updating tox file for correct GitHub actions check
    - Adding supported files

### CODE REVIEW

**Comment by jalam453** [comment]
GitHub actions checks commit

File: docs/make.bat [hunk 1]
```diff
@@ -0,0 +1,36 @@
+@ECHO OFF
```
Comment (midnightercz) [inline docs/make.bat:1]: Do we even need this file?

File: docs/source/README.rst [hunk 2]
```diff
@@ -0,0 +1,2 @@
+# pubtools-content-gateway (CGW)
+# A tool to do pushes on CGW
```
Comment (midnightercz) [inline docs/source/README.rst:2]: I guess this should be expanded. At least put there link to the github pages

File: docs/source/conf.py [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+
+
+# -- Project information -----------------------------------------------------
+
+project = "pubtools-content-gateway"
+copyright = "2022, Red Hat"
+author = "Red Hat"
+
+# The short X.Y version
+version = ""
```
Comment (midnightercz) [inline docs/source/conf.py:28]: This shouldn't be empty

File: docs/source/conf.py [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+# The full version, including alpha/beta/rc tags
+release = ""
```
Comment (midnightercz) [inline docs/source/conf.py:30]: This one as well

File: pubtools/_content_gateway/rough.py [hunk 5]
```diff
@@ -0,0 +1,59 @@
+# from utils import yaml_parser
```
Comment (midnightercz) [inline pubtools/_content_gateway/rough.py:1]: ???

**Review by midnightercz — CHANGES_REQUESTED** [review]

File: pubtools/_content_gateway/rough.py (hunk 5 above)
Comment (jalam453) [reply pubtools/_content_gateway/rough.py:1]: Sorry, my bad. This file shouldn't be added.

File: docs/source/conf.py (hunk 4 above)
Comment (jalam453) [reply docs/source/conf.py:30]: sure I'll update.

File: docs/source/conf.py (hunk 3 above)
Comment (jalam453) [reply docs/source/conf.py:28]: Ack

**Comment by jalam453** [comment]
Closing this MR. Submitted a new one.
https://github.com/release-engineering/pubtools-content-gateway/pull/14

**Pull request closed without merging**